cli/running/instance.go

```diff
@@ -104,9 +104,26 @@ func (inst *Instance) Start() error {
 		return err
 	}
 	inst.Cmd.Env = append(os.Environ(), "TT_CLI_INSTANCE="+inst.appPath)
-	inst.Cmd.Env = append(inst.Cmd.Env,
-		"TT_CLI_CONSOLE_SOCKET="+inst.consoleSocket)
 
+	// It became common that console socket path is longer than 108 symbols(sun_path limit).
+	// To reduce length of path we use relative path
+	// with chdir into a directory of console socket.
+	// e.g foo/bar/123.sock -> ./123.sock
+	if inst.consoleSocket != "" {
+		if len("./"+filepath.Base(inst.consoleSocket)) > 108 {
+			return fmt.Errorf("socket name is longer than 106 symbols: %s",
```

Comment:
106->108, Use constant var.

Reply:
108 on Linux, 104 on Mac OS AFAIR. But that's with leading `'\0'`. So 107 and 103.

Reply:
See also a remark regarding using 108th and 104th byte in https://github.com/tarantool/tarantool/issues/4634 and links regarding Mac OS.

Reply:
from unix(7):
The length of the pathname, including the terminating null byte, should not exceed the size of sun_path.

Reply:
Done